src/scss/main.scss

```diff
@@ -1,99 +1,95 @@
 // Make sure the charset is set appropriately
 @charset "UTF-8";
-
 // Foundation Components
-@import
-"../../node_modules/foundation-apps/scss/foundation";
-
+@import "../../node_modules/foundation-apps/scss/foundation";
```

Comment:
We should be able to configure gulp-sass to look in our node_modules folder to find foundation stuff. Then we won't need the ugly ```../../node_modules/``` stuff